server/athenian/api/models/web/base_model_.py

```diff
@@ -26,6 +33,11 @@ def __new__(mcs, name, bases, dikt, sealed=True):
             attribute_map.update(dikt["attribute_map"])
```

Comment:
missing: `default_values.update(dikt["default_values"])`

Reply:
done, anyway we could also evaulate to drop support for dikt['default_values'] and dikt['attribute_types'], annotations are enough

Reply:
They are quite useful. In fact, dataclasses have their own version in `__dataclass_fields__`